apiclient/http.py

```diff
@@ -1113,7 +1112,9 @@ def _serialize_request(self, request):
       msg['content-length'] = str(len(request.body))
 
     # Serialize the mime message.
-    fp = io.StringIO()
+    # TODO: Use io.StringIO after we update it to support only Py3
```

Comment:
can we add a comment with `py2to3` as a tag, we can use this to identify all the todo items after we have finished all our work related to the upgrade. This will help us in identifying the pending work when we upgrade to Python 3.

Also I would recommend to add a conditional import here along with a comment to cleanup the code when we run Python 3. By doing so, we will reduce our work in the future, we only need to remove a condition check which is lot simpler than add appropriate import when we finally switch to Python 3. We can even remove all these kind of conditions AFTER we upgrade to Python 3.

I suggest we should do what is in the StackOverflow answer, that way later on when we switch to Python 3 we can simply remove the try except block.

Reply:
Cool.. This is done.